doc/Development.md

```diff
@@ -0,0 +1,80 @@
+# Development instructions
+
+## Introduction
+
+Tauntaun consists of two main parts: a backend implemented in Python and a frontend implemented in React/TypeScript. The backend uses the Quart server framework for network I/O. It uses PyDCS to load/save DCS .miz files as well as to provide abstractions for DCS mission file state. The frontend is implemented in React and uses (react-)esri-leaflet to provide mapping services.
+
+## TL;DR
+
+To get up and running, you need to install [Poetry](https://python-poetry.org/) and node/yarn. Then run the following commands
+
+```
```

Comment:
Missing "git submodule update --init --recursive".

Reply:
Thanks

Reply:
Fixed via removing the submodule altogether 😛